opsdroid/parsers/regex.py

```diff
@@ -1,16 +1,16 @@
 """A helper function for parsing and executing regex skills."""
 
 import logging
-import re
+import regex
```

Comment:
Just a tiny typo here, it needs to be back to re.
 
```suggestion
import re
```

Reply:
Hi @FabioRosado 

Thanks for your message. Yes i used opsdroid and never have to deal with identical named group need until now :-)

I used the regex module from [pypi regex](https://pypi.org/project/regex/) and used it after the **import regex** instead of **re**

Reply:
Hi @FabioRosado 

an update please if it's possible

Thanks

Reply:
This is fine. This PR is intentionally swapping our the standard `re` library with the third party `regex` library from PyPI.